docs/spec_draft.md

```diff
@@ -1243,6 +1244,78 @@ Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.slice
+
+### Semantics
+
+Extracts a sub-tensor from the `operand` using the dimension indices from the
+operands.
+
+More formally, `result[i0, ..., ik, ..., iR-1] = operand[j0+s0, ..., jk+sk, ..., jR-1+sR-1]`
+for all `sk` in `strides` for all `jk` in `start_indices[k]` $\leq$ `jk` $\lt$
+`limit_indices[k]` where 0 $\leq$ `k` $\lt$ rank(`operand`).
+
+Shape is calculated by `dim(result, i)` = $\lceil$`(limit_indices[i]-start_indices[i])/stride[i]`$\rceil$
+for `i`th dimension size of `operand`.
```

Comment:
of `result`?

Reply:
Sorry, it should say:
```
`i`th dimension of `operand`.
```
But since `operand` and `result` have the same rank, both are interchangeable. I used `operand` here since we are indexing using the `operand` to get the `result`.

Reply:
I  see you point. You are right.

My suggestion was because we are defining the dimension size at ith axis for result, thus to me it makes more sense to "apply" `i` in the context of result.